.github/workflows/default.yml

```diff
@@ -0,0 +1,24 @@
+name: default build
+
+on:
+  push:
+    branches:
+      - master
+    tags:        
+      - '*'

```

Comment:
@onukura  
Is this workflow for `latest` tag? 
https://github.com/docker/build-push-action#tag_with_ref

Then, this definition means that it will be triggered when you `git push` to master branch of gitbucket/gitbuclet-docker not gitbucket/gitbucket?

Reply:
@SIkebe Yes. it means push to gitbucket/gitbucket. otherwise if it is only `tags: - '*'`, `latest` image will not be published... So I   included master branch push trigger just for `latest` image.

Reply:
I mean "`git push` to master branch of **gitbucket/gitbuclet-docker**"

Reply:
Sorry. I misspelled. Yes push to **gitbucket/gitbuclet-docker**

Reply:
Hmm, `latest` is a bit unclear to me in this case... I'm worried that developers using Docker Hub would be under the impression that `latest` is equivalent to the **latest release** (i.e. 4.34.0) or **master branch of gitbucket/gitbucket**. 
I think release tags and nightly builds are enough for our use cases.

@takezoe What do you think?

Reply:
Ah, I figured it out. Though `latest` will be pushed when you `git push` to master of gitbucket/gitbucket-docker, the version of gitbucket.war is hard-coded in `Dockerfile`. So it's the same thing as "the latest release in gitbucket/gitbucket".
@onukura Sorry for confusing you!

Reply:
@SIkebe Yeah! Even if you `git push` to gitbucket/gitbucket-docker many time in order  to edit `README.md` for example, Docker image with `latest` tag does not change because it is hard-coded.

Reply:
@onukura Just to confirm, does this mean that the latest and each release tag will be remain and `4` tag will be discontinued?

Reply:
Ah, you have mentioned about available tags here: https://github.com/gitbucket/gitbucket-docker/issues/22#issuecomment-673522680